### DISCUSSION TITLE
Whitelist option

### DISCUSSION THREAD

**Discussion by sohate** [aleppo-io/last-window-quits#22] [Feature requests]
Hey! Thanks for app. The app really needs a list of apps to work with. I mean right now we have only Blacklist, but it is hard to add all my apps to it, because I need your app only for a few apps. So, can you add an something like Blacklist but reverse.

Sorry, english is not my native language.

**Comment by lawand-dot-io** [comment]
Hello. What you're talking about is a whitelist. Some other users have also mentioned this so I'll consider adding this option. Cheers
